.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules/
 dist/
 npm-debug.log
 yarn-error.log
+coverage/
```

Comment:
I prefer to have everything about testing under the `test`-directory, so if it's possible to change this, please do so 😄